gnomad_constraint/utils/constraint.py

```diff
@@ -734,6 +732,67 @@ def calculate_mu_by_downsampling(
     return annotate_mutation_type(ht)
 
 
+def add_oe_lof_upper_rank_and_bin(
+    ht: hl.Table, use_mane_select_instead_of_canonical: bool = True
```

Comment:
this is another nitpick, but this name (`use_mane_select_instead_of_canonical`) is a little misleading, since this will also use the canonical annotation for transcripts not yet included in MANE select